src/Components/Components/src/Routing/NotFoundEventArgs.cs

```diff
@@ -9,16 +9,7 @@ namespace Microsoft.AspNetCore.Components.Routing;
 public sealed class NotFoundEventArgs : EventArgs
 {
     /// <summary>
-    /// Gets the path of NotFoundPage.
+    /// Gets the path of NotFoundPage. If the path is set, it indicates that the router has handled the rendering of the NotFound contents.
```

Comment:
Do not mention the router explicitly. Instead use a general term like "A subscriber"